modules/bubblewrap.nix

```diff
@@ -75,7 +75,7 @@ in {
 
     env = mkOption {
       description = "Environment variables to set.";
-      type = with types; attrsOf (nullOr str);
+      type = with types; attrsOf (oneOf [ (nullOr str) slothValue ]);
```

Comment:
I *think* this could just be `nullOr sloth.type`.

Reply:
I tried but it doesn't work for all strings, only paths, per sloth.type.check.

Reply:
Right, I guess sloth types should support any strings at this point, not just paths.